src/sql/workbench/browser/modelComponents/checkbox.component.ts

```diff
@@ -20,7 +20,7 @@ import { IWorkbenchThemeService } from 'vs/workbench/services/themes/common/work
 @Component({
 	selector: 'modelview-checkbox',
 	template: `
-		<div #input [style.width]="getWidth()" [style.height]="getHeight()"></div>
```

Comment:
if the width and height is removed, how do we control the width and height for checkbox?

Reply:
It's passed to the checkbox control already when the properties are set

https://github.com/microsoft/azuredatastudio/blob/master/src/sql/workbench/browser/modelComponents/checkbox.component.ts#L85-L89